test/tests

```diff
@@ -5,30 +5,84 @@ source "$( pwd )/test/helpers"
 
 test::detect_legacy_default() {
     # Test the buildpack detect script with a single simple .php file
-    test::helpers::detect "legacy_default"
-    assertCapturedSuccess
-    assertCapturedEquals "PHP (classic)"
+    test::utils::detect "legacy_default"
+    test::utils::assertCapturedSuccess
+    test::utils::assertCapturedEquals "PHP (classic)"
 }
 
 test::compile_legacy_default() {
     # Test the buildpack compile script with a single simple .php file
-    test::helpers::compile "legacy_default"
-    assertCapturedSuccess
+    test::utils::compile "legacy_default"
+    test::utils::assertCapturedSuccess
+
+    test::utils::resetCapture
+
+    test::helpers::get_php_version
+    test::utils::assertCapturedStartswith "PHP 8.1."
 }
 
 test::detect_composer_default() {
-    test::helpers::detect "composer_default"
-    assertCapturedSuccess
-    assertCapturedEquals "PHP (composer.json)"
+    test::utils::detect "composer_default"
+    test::utils::assertCapturedSuccess
+    test::utils::assertCapturedEquals "PHP (composer.json)"
 }
 
 test::compile_composer_default() {
-    test::helpers::compile "composer_default"
+    test::utils::compile "composer_default"
     # We can't use assertCapturedSuccess here:
     # Even with an empty composer.json file, composer will use stderr to warn
-    # us that nothing had to be installed -_-, causing the test to fail for no
-    # reason.
-    assertSuccess
+    # us that nothing had to be installed, causing the test to fail for no
+    # reason -_-
+    test::utils::assertSuccess
+
+    test::utils::resetCapture
+
+    test::helpers::get_php_version
+    test::utils::assertCapturedStartswith "PHP 8.1."
+}
+
+test::compile_composer_php80() {
+    # PHP 8.0 is only available on stack `scalingo-20`:
+    [[ "${STACK}" != "scalingo-20" ]] && startSkipping
```

Comment:
question: shouldn't we display a message in this case to make sure the developer understand what happens?

Reply:
The `startSkipping` function will only support this feature in the (maybe-coming-one-day-) `2.1.9` release (see https://github.com/kward/shunit2/issues/145).

I'll mimic the behavior in the following update :smiley: